tools/docker/dev_container.Dockerfile

```diff
@@ -10,8 +10,9 @@ RUN pip install --default-timeout=1000 $TF_PACKAGE==$TF_VERSION
 
 COPY tools/install_deps /install_deps
 COPY requirements.txt /tmp/requirements.txt
-RUN pip install -r /install_deps/black.txt \
-    -r /install_deps/flake8.txt \
+RUN pip install \
+    # -r /install_deps/black.txt \
```

Comment:
The yapf and clang-format should be installed.

Reply:
As regular dev container, some software also need to install on it, refer to: https://github.com/tensorflow/recommenders-addons/blob/master/tools/docker/cuda11.2-cudnn8.1-ubuntu18.04-manylinux2010-multipython.Dockerfile#L48
Tomorrow I will do this.